new/t2.js

```diff
@@ -0,0 +1,25 @@
+/*eslint no-debugger: "error"*/
+
+function isTruthy1(x) {
+    debugger;
+    return Boolean(x);
+}
+
+function isTruthy2(x) {
+    debugger;
+    return Boolean(x);
+}
+
+var x = 5;;
+
+if (!!foo2) {
+    // ...
+}
+
+if (Boolean(foo)) {
```

Comment:
Redundant Boolean call.

```suggestion
if (foo) {
```